demos/common/monitors/memory_monitor.cpp

```diff
@@ -0,0 +1,258 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "memory_monitor.h"
+
+#ifdef WIN32
+#include <algorithm>
+
+#define PSAPI_VERSION 1 // for psapi
```

Comment:
Given that we certainly don't support versions of Windows older than 7, why not define it to 2? Then you'll no longer have to link to `psapi.dll`.

Reply:
redefined